src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy

```diff
@@ -178,6 +178,31 @@ class TweetContentGeneratorSpec extends Specification {
         posts << [posts(), postsWithLongHandles()]
     }
 
+    def "should have just two handles if twitter is missing for a third one"() {
+        given:
+        NewsletterIssue issue = NewsletterIssue
+            .builder()
+            .issueNumber(ISSUE_NUMBER)
+            .heading("issue heading")
+            .blogPosts(blogWithoutTwitter())
+            .build()
+
+        when:
+        String tweetContent = contentGenerator.generateTweetContent(issue)
+
+        then:
+        def handles = /.*m\.in\. @personal\d{1} i @company\d{1}.*/
```

Comment:
`personal1` and `company1` are duplicated here and below in the `blogWithoutTwitter()` method

Reply:
The same story is for multiple other specs. What other would you expect here?